### ISSUE TITLE
Brakeman cannot parse ruby 2.6's endless range syntax for arrays

### ISSUE THREAD

**Issue by blaedj** [presidentbeef/brakeman#1325]
### Background

Brakeman version: brakeman 4.4.0
Rails version: Rails 6.0.0.beta2
Ruby version: ruby 2.6.1p33 (2019-01-30 revision 66950) [x86_64-darwin18]

Link to Rails application code: https://github.com/blaedj/brakemantest

#### Parse Error

```
Error: /Users/blaed/code/ruby/brakemantest/app/models/book.rb:3 :: parse error on value "]" (tRBRACK)
Location: Could not parse /Users/blaed/code/ruby/brakemantest/app/models/book.rb
```

Minimal example that does not parse (see also https://github.com/blaedj/brakemantest/blob/master/app/models/book.rb)

```ruby
class Book
  def scanThings(arr)
    arr[1..].each do |i|
      puts i
    end
  end
end

```

**Comment by presidentbeef** [comment]
Hi @blaedj,

Yes, this will be added in the next release of RubyParser.

**Comment by blaedj** [comment]
great thanks!

**Comment by presidentbeef** [comment]
This is fixed in with ruby_parser 3.13.0 with is included in the Brakeman 4.5.0 release.
